tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -11,12 +11,13 @@ internal static class Program
     private static unsafe void Main(string[] args)
     {
         Add(1, 2);
-        var tempObj = new TestClass(1337);
+        TestClass tempObj = new TestDerivedClass(1337);
         int tempInt = 0;
         (*(&tempInt)) = 9;
     
         tempObj.TestMethod("Hello");
-
+        tempObj.TestVirtualMethod("Hello");
```

Comment:
In testing, I noticed that this now prints before "Hello from C#". It's ok for this change, but we should try to remember to move them lower.